Update storage-troubleshoot-windows-file-connection-problems.md

jboeshart opened the pull request:
Added a note to the service account section to add the credential to Credential Manager and to not use a mapped drive letter.

azurecla commented:
Hi __@stuartozer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (stuarto). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jboeshart committed:
Update storage-troubleshoot-windows-file-connection-problems.md

Added a note to the service account section to add the credential to Credential Manager and to not use a mapped drive letter.
